README.md

```diff
@@ -338,3 +338,7 @@ I hope to improve this algorithm to be faster in the next few releases.
 ## Important Links
 
   * [Documentation](http://hexdocs.pm/beaker)
+
```

Comment:
Maybe let's make a link here to `API_doc.md` so people can easily view it from Github, or do you think it'll be used internal only?